README.md

```diff
@@ -18,16 +17,16 @@ For more information about up-to-date syntax and language features, check out th
 > **🌶 The language is still going through its early iterations.**  
 > To build the compiler from source, you will need to:
 >
-> - [Set up](https://github.com/llvm/llvm-project/releases/tag/llvmorg-12.0.1) LLVM 12 on your machine (I'm working on including LLVM as part of the project).
-> - [Install](https://www.rust-lang.org/learn/get-started) the rust compiler
+> - [Set up](https://github.com/llvm/llvm-project/releases/tag/llvmorg-12.0.1) LLVM 12 on your machine.You should build from source under windows.
+> - [Install](https://www.rust-lang.org/tools/install) the rust compiler
 
 As there's no freestanding binary yet, we still need to build the compiler from source. To build the compiler, run:
 
 ```
 cargo build
 ```
 
-**Note:** Although the compiler itself supports both Windows and Linux, the standard library is very young, and doesn't support Windows yet. This will be implemented in the near future. Have a look at the [Tasks](#Tasks) section.
+**Note:** Although the compiler itself supports both Windows and Linux(Mac OS support is on the way), the standard library is very young, and doesn't support Windows yet. This will be implemented in the near future. Have a look at the [Tasks](#Tasks) section.

```

Comment:
```suggestion
**Note:** Although the compiler itself supports both Windows and Linux (Mac OS support is also planned), the standard library is very young, and doesn't support Windows yet. This will be implemented in the near future. Have a look at the [Tasks](#Tasks) section.

```